tests/wicked/aggregate/sut/t10_bonding_ab_ping_2_ips.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Bonding, Active-Backup Ping 2 IPs
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+
+
+sub run {
+    my ($self, $ctx) = @_;
+    record_info('INFO', 'Bonding, Active-Backup Ping 2 IPs');
+    $self->setup_bond('alb', $ctx->iface(), $ctx->iface2());
+    $self->validate_interfaces('bond0', $ctx->iface(), $ctx->iface2());
+    my $active_link1 = $self->get_bond_active_link('bond0');
+    sleep 5;    # Wait some seconds until bond process fails over to the other interface
```

Comment:
any chance to replace it with some loop where we actually check for something . Such "blind" timeout tend to grow over the time ...

Reply:
Sure, I can implement a loop with timeout, it's cleaner. I just went for the quick way after doing some manual test and realizing that the failover is quick (<1 sec)

Reply:
sure things , but this "<1 sec" could easily go to 10 seconds in some busy worker running 10+ jobs simultaneously.

Reply:
done